ods_ci/tests/Tests/400__ods_dashboard/415__ods_dashboard_projects/415__ods_dashboard_projects.robot

```diff
@@ -253,6 +253,28 @@ Verify User Can Delete A Data Connection
     Data Connection Should Not Be Listed    name=${DC_3_S3_NAME}
     Check Data Connection Secret Is Deleted    dc_name=${DC_3_S3_NAME}    namespace=${ns_name}
 
+Verify user can create a workbench with an existing data connection
+    [Tags]  Tier1  ODS-2176
+    [Documentation]  Verifies users can create a workbench with an existing data connection
+
+    Open Data Science Project Details Page       project_title=${PRJ_TITLE}
+    Create S3 Data Connection  project_title=${PRJ_TITLE}
+    ...                    dc_name=${DC_S3_NAME}
```

Comment:
I think we should rather use a different name for this data connection - because otherwise we need to be sure that at the end of this test and also at the end of the `Verify User Can Create A S3 Data Connection And Connect It To Workbenches` test this data connection is cleaned. Otherwise these will conflict.

Reply:
name changed